test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/findModelProperty.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var chai = require('chai');
```

Comment:
var expect = require('chai').expect;

Ideally it should be moved out to `test/helpers/expect.js` and simply `var expect = require('../helpers/expect');` here.